packages/block-library/src/navigation/index.php

```diff
@@ -471,7 +471,7 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 	// - we don't have a relationship to a `wp_navigation` Post (via `ref`).
 	// ...then create inner blocks from the classic menu assigned to that location.
 	if (
-		defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN &&
+		apply_filters( 'gutenberg_block_core_navigation_use_interactivity_api', true ) &&
```

Comment:
Is this right? The code in here is not all related to Interactivity API.

Reply:
You're right, but let's use your PR instead: https://github.com/WordPress/gutenberg/pull/52579